### ISSUE TITLE
GIB Issue / New Minor Forcing

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#408]
Last tournament on BBO BEN misdefended as it could not solve this bidding
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/7affc971-01aa-4602-8c54-d25703ee0eb4)

When playing New Minor Forcing, i think you also play these BAR-bids, showing 6-card minor and 4-card major. Not invitational, but just to play.

After 
1C-(P)-1S-(P)-
1N-(P) it would be nice with this implementation

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/bdeef7e6-3047-4982-a210-169ed5aeaa5e)

Comparing to BBA I can see the following difference
2C should be 5+C
2H is not forcing, so limited to 11 HCP
2N is 11-12 HCP
3H is 5-5 but only invitational
3S is invitational

**Comment by EdwardPiwowar** [comment]
Is E after pass for GIB meaning?

**Comment by ThorvaldAagaard** [comment]
I do not understand the question

**Comment by EdwardPiwowar** [comment]
Is it the same?
https://bridgewinners.com/article/view/what-does-this-bid-mean-2-k07n7nru4j/

**Comment by ThorvaldAagaard** [comment]
1Mi - 1M
1N - 3OMi should be 4-6 and to play if that is the question
It is called a BAR-bid

**Comment by EdwardPiwowar** [comment]
This means that BAR-bid should be added as a convention.

**Comment by ThorvaldAagaard** [comment]
Yes, that could be a solution, as NMF not has a clear definition of how it is defined. 
I assume it is the same for 2-way NMF and checkback stayman
